src/test/java/uk/gov/justice/hmpps/offenderevents/subscribe/PrisonerEventsListenerTest.java

```diff
@@ -0,0 +1,67 @@
+package uk.gov.justice.hmpps.offenderevents.subscribe;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.json.JsonTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import uk.gov.justice.hmpps.offenderevents.model.OffenderEvent;
+import uk.gov.justice.hmpps.offenderevents.services.HMPPSDomainEventsEmitter;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.mockito.Mockito.verify;
+
+@JsonTest
+class PrisonerEventsListenerTest {
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @MockBean
+    private HMPPSDomainEventsEmitter eventsEmitter;
+
+    @Captor
+    private ArgumentCaptor<OffenderEvent> offenderEventArgumentCaptor;
+
+    private PrisonerEventsListener listener;
+
+    @BeforeEach
+    void setUp() {
+        listener = new PrisonerEventsListener(objectMapper, eventsEmitter);
+    }
+
+    @SneakyThrows
```

Comment:
Don't see that very often! What does it mean on a test method?

Reply:
`listener.onPrisonerEvent` throws an exception, Intellij suggested either adding that exception to the test method of sneak throws, I chose the later for the grins